README.md

```diff
@@ -7,3 +7,36 @@
 ```
 mvn clean install
 ```
+
+## Usage
+
+### Dependency Check (OWASP)
+
+Clients using the Ingest API generated client code, or server code using the generated stubs,
+can add the following to their `dependency-check-maven` configuration to inherit the
+suppressions and avoid having to repeat them in their own project.
+
+_Maven_
+
+```xml
+<build>
+    <plugins>
+        <plugin>
+            <groupId>org.owasp</groupId>
+            <artifactId>dependency-check-maven</artifactId>
+            <configuration>
+                <suppressionFiles>
+                    <suppressionFile>owasp-suppressions.xml</suppressionFile>
+                </suppressionFiles>
+            </configuration>
+            <dependencies>
+                <dependency>
+                    <groupId>com.connexta.ingest</groupId>
+                    <artifactId>ingest-api-cxf-client</artifactId>
+                    <version>${ingest-api.version}</version>
+                </dependency>
+            </dependencies>
+        </plugin>
+    </plugins>
+</build>
```

Comment:
❓ Can you give an example for how to handle a client depending on 2 APIs
❓ What about those using gradle?

Reply:
Moved the suppression files under a directory that has the same as the artifact ID.
I haven't tried Gradle yet. I'll update if we all decide this is a viable approach.